### DISCUSSION TITLE
Make Raw WPM More Accurate

### DISCUSSION THREAD

**Discussion by TKelsier** [monkeytypegame/monkeytype#1056] [Ideas] [answered]
Currently Raw WPM doesn't take into account all characters typed regardless of it being correct or not and only counts incorrect characters that are not fixed, therefore you can type a word perfectly, delete it, type it again, then finish the test and you'll only get counted for typing the word once.  While this should be the case for the true wpm, raw wpm should reflect the repeated typing. 

The following is a video example of the issue:
https://user-images.githubusercontent.com/69769319/110225173-532d1100-7ea8-11eb-82b9-630dd8aee5ad.mp4

As you can see, in the video I type the first word multiple times however the end total characters typed is only 49 and the raw and normal wpm are the same.

**Comment by Miodec** [comment, marked as answer]
Yeah, looks like the graph is using the correct data but the actual number is not. Added #1060
